lib/Doctrine/DBAL/Query/QueryBuilder.php

```diff
@@ -483,6 +484,15 @@ public function select($select = null/*, string ...$selects*/)
             return $this;
         }
 
+        if (is_array($select)) {
+            Deprecation::triggerIfCalledFromOutside(
```

Comment:
`QueryBuilder` isn't used by the DBAL, its API is only called from tests, should be `trigger()`.